### PR TITLE
[DEV 7913] Left-align Covid Profile headings & descriptions

### PR DESCRIPTION
**Pull request by afnazu**
**High level description:**

For each section heading of the Covid profile that is currently center-aligned, the text is updated to match the latest mockup (left-aligned) 

*note this is prep work for ARP branding ticket https://federal-spending-transparency.atlassian.net/browse/DEV-7877

**JIRA Ticket:**
[DEV-7913](https://federal-spending-transparency.atlassian.net/browse/DEV-7913)

**Mockup:**
https://bahdigital.invisionapp.com/share/4HIAGPS7T3X#/screens

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [n/a] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [n/a] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [n/a] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [n/a] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [n/a] Design review complete `if applicable`
- [n/a] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/covid19/index.scss [hunk 1]
```diff
@@ -170,10 +163,9 @@
 
         .heading__container {
           border-radius: 0.5rem 0.5rem 0 0;
-          @include display(flex);
+          @include display(block);
           flex-wrap: wrap;
```
Comment (brett-varney) [inline src/_scss/pages/covid19/index.scss:167]: This should be removed; no longer a flex container

**Review by brett-varney — CHANGES_REQUESTED** [review]
remove old css rule, and AC3 not met (different from mock)

File: src/_scss/pages/covid19/index.scss (hunk 1 above)
Comment (brett-varney) [reply src/_scss/pages/covid19/index.scss:174]: @afnazu Did you see this line should be removed?

Comment (afnazu) [reply src/_scss/pages/covid19/index.scss]: @brett-varney Thanks, removed css rule and updated footer styling

**Review by brett-varney — APPROVED** [review]

**Pull request merged**